client/verta/verta/_internal_utils/model_dependencies.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+
+import ast
+import inspect
+import textwrap
+from types import ModuleType
+from typing import Callable, get_type_hints, List, Tuple, Type
+
+from verta.registry import VertaModelBase
+
+
+def list_class_functions(model_class: Type[VertaModelBase]) -> List[Tuple[str, Callable]]:
```

Comment:
Not a big deal, but we might want to make all our `list_*()` functions return the same format; this returns both function names and objects, but the others return just the module names.

Reply:
Changed return type to be more consistent (now just a set of callables)